main.rb

```diff
@@ -0,0 +1,42 @@
+# program takes user input to add planets & list planets
+require_relative 'solar_system.rb'
+def user_loop
+  solar_system = SolarSystem.new('Sol')
+  puts "Would you like to add planets? "
+  print "  y - add_planets
+  n - exit \n"
+  print "Type y or n: "
+  answer = gets.chomp
+  while answer == "y"
```

Comment:
This method would be better if it gave you some kind of interactive menu to work from adding, and viewing planets.